django.Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM python:3.13
 ENV PYTHONUNBUFFERED=1
-WORKDIR /app
-COPY . /app
 
 RUN apt-get update -y  \
     && apt-get --no-install-recommends install -y  \
         gettext  \
         libldap2-dev  \
         libsasl2-dev  \
         postgresql-client  \
-        postgresql-client-common \
-    && rm -rf /var/lib/apt/lists/*  \
-    && pip install poetry  \
-    && poetry install  \
+        postgresql-client-common  \
+        python3-poetry  \
```

Comment:
I would prefer installing poetry from pip or maybe even directly from source. This way, we can control the version since it had heavy breaking changes recently